src/metabase/lib/schema/expression/temporal.cljc

```diff
@@ -7,7 +7,14 @@
    [metabase.lib.schema.literal :as literal]
    [metabase.lib.schema.mbql-clause :as mbql-clause]
    [metabase.lib.schema.temporal-bucketing :as temporal-bucketing]
-   [metabase.util.malli.registry :as mr]))
+   [metabase.util.malli.registry :as mr])
+  #?@
+  (:clj
+   [(:import
+     (java.time ZoneId))]
+   :cljs
+   [(:require
+     ["moment" :as moment])]))
```

Comment:
I copied over the valid-timezone-check stuff from the old Schema version of the schema for legacy MBQL